packages/geocoder/src/apis/otp/index.ts

```diff
@@ -0,0 +1,51 @@
+
+// eslint-disable-next-line prettier/prettier
+import type { AutocompleteQuery  } from "../../geocoders/types"
+
+type FetchArgs = {
+  url: string
+  query: string
+}
+
+type OTPGeocoderResponse = {
+  results: {
+    lat: number,
+    lng: number,
+    description: string,
+    id: string
+  }[]
+}
+
+
+function run({  query, url }: FetchArgs): Promise<OTPGeocoderResponse> {
```

Comment:
Maybe we should specify in the return type that it's possible to get undefined as a response, which will help enforce null coalescing operators when the function is used.